backend/app/routers/auth.py

```
@@ -33,10 +39,18 @@ def get_db():
 
 def authenticate_user(username: str, password: str, db: Session) -> Users | bool:
     user = db.query(Users).filter(Users.username == username).first()
+    print(f"[DEBUG] authenticate_user: username={username}")
+    print(f"[DEBUG] User found: {user is not None}")
     if user is None:
+        print(f"[DEBUG] User not found in database")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)